.github/labels.yml

```diff
@@ -1,111 +1,81 @@
 # Type
-- name: Type:Bug
-  color: d93f0b
+# uses less saturated colors
+- name: bug
+  color: faadaf
   description: "Something is broken"
 
-- name: Type:Needs Triage
-  color: bebaf4
-  description: "Awaiting triage"
-
-- name: Type:Needs Attention
-  color: e58b7e
-  description: "It's been a while since this was pushed on"
-
-- name: Type:Needs Info
-  color: bebaf4
-  description: "Needs further information"
-
-- name: Type:Discussion
+- name: discussion
   color: bebaf4
   description: "Discussing a topic with no specific actions yet"
 
-- name: Type:Feature
-  color: 0052cc
+- name: feature
+  color: b0f0fa
   description: "Something is missing"
 
-- name: Type:Hygiene
-  color: bebaf4
-  description: "Code quality and project level things"
-
-- name: Type:Testing/CI
-  color: f4e49a
-  description: "Unit tests and/or continuous integration"
-
-# Area
-- name: Area:Array
-  color: 006b75
-  description: ""
-
-- name: Area:Bag
-  color: 0052cc
-  description: ""
-
-- name: Area:Config
-  color: 55b3c1
-  description: ""
-
-- name: Area:Core
-  color: "000000"
-  description: ""
+- name: documentation
+  color: f9d0c4
+  description: "Improve or add to documentation"
 
-- name: Area:Daraframe
-  color: fbca04
-  description: ""
+- name: hygiene
+  color: b5ccfa
+  description: "Improve code quality and reduce maintenance overhead"
 
-- name: Area:Delayed
-  color: 4f6edd
-  description: ""
+- name: deprecation
+  color: f4e49a
+  description: "Something is being removed"
 
-- name: Area:Documentation
-  color: f9d0c4
-  description: ""
+- name: tests
+  color: a0f9b4
+  description: "Unit tests and/or continuous integration"
 
-- name: Area:IO
-  color: e99695
-  description: ""
+# Status
+# prefixed with "needs"
+- name: needs attention
+  color: 6d626c
+  description: "It's been a while since this was pushed on. Needs attention from the owner or a maintainer."
 
-- name: Area:Scheduler
-  color: 64db8e
-  description: ""
+- name: needs review
+  color: 8c918c
+  description: "Needs review from a contributor."
 
-- name: Area:Diagnostics/Dashboard
-  color: ffcce7
-  description: ""
+- name: needs info
+  color: d0cfcc
+  description: "Needs further information from the user"
 
-- name: Area:ML
-  color: 77dd8d
+# Area
+# more saturated colors, projects are encouraged to add many more of these
+- name: config
+  color: 1586c0
   description: ""
 
-- name: Area:Deployment
-  color: 6eaadd
+- name: core
+  color: "000000"
   description: ""
```

Comment:
What is core?

Reply:
That's an existing label in dask https://github.com/dask/dask/labels/core but I'm not sure if it overlaps too much with the new labels (like maybe `hygiene`)

Reply:
I believe this is label was for low-ish level issues that impact a large part of the `dask/dask` codebase (e.g. graph optimization is a topic which applies to arrays, dataframes, bags, and delayed). Though I agree "core" is a vague term. I don't have strong feelings on keeping it or dropping it. We can always iterate later.